src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -4022,17 +4022,23 @@ public override IOperation VisitCaughtException(ICaughtExceptionOperation operat
 
         public override IOperation VisitInvocation(IInvocationOperation operation, int? captureIdForResult)
         {
-            if (operation.Instance != null)
+            (IOperation visitedInstance, ImmutableArray<IArgumentOperation> visitedArguments) = VisitInstanceWithArguments(operation.Instance, operation.Arguments);
+            return new InvocationExpression(operation.TargetMethod, visitedInstance, operation.IsVirtual, visitedArguments, semanticModel: null, operation.Syntax,
+                                            operation.Type, operation.ConstantValue, IsImplicit(operation));
+        }
+
+        private (IOperation, ImmutableArray<IArgumentOperation>) VisitInstanceWithArguments(IOperation instance, ImmutableArray<IArgumentOperation> arguments)
+        {
+            if (instance != null)
             {
                 // PROTOTYPE(dataflow): drop instance for a static method
```

Comment:
>// PROTOTYPE(dataflow): drop instance for a static method [](start = 16, length = 57)

Please implement this work item and add relevant tests

Reply:
Done - I have pushed a new commit addressing the prototype comments and adding relevant unit tests for all affected operation nodes.

---
In reply to: [187757013](https://github.com/dotnet/roslyn/pull/26797#discussion_r187757013) [](ancestors = 187757013)